src/components/Card.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Card.propTypes = {
+  image: PropTypes.node,
+  title: PropTypes.string,
+  ratingimage: PropTypes.node,
+  text: PropTypes.string,
+}
+
+export default function Card({ image, title, ratingimage, text }) {
+  return (
+    <Wrapper>
+      <img src={image} alt="" />
+      <div>
+        <h2>{title}</h2>
+        <img src={ratingimage} alt="" />
```

Comment:
Does `ratingImage` contain information about how the location is rated? Is there a rating displayed? If so, then the information should be provided in the `alt` attribute.